roles/openshift_health_checker/openshift_checks/diskspace_availability.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DiskAvailability(OpenShiftCheck):
+    """Check that recommended disk space is available."""
+
+    name = "disk_availability"
+    tags = ["preflight"]
+
+    recommended_node_space_gb = 15.0
+    recommended_master_space_gb = 40.0
+    minimum_docker_space_percent = 5.0
```

Comment:
These should be configurable in run time.

They may be defined as variables in the `preflight/check.yml` playbook, and their names should have some prefix to avoid name collisions...